lib/actions.ts

```diff
@@ -71,21 +71,43 @@ export async function createPassport(formData: FormData) {
 		trueFirstName,
 		trueDateOfBirth,
 		trueDateOfIssue,
+		trueCeremonyTime,
 		placeOfOrigin,
 		stringUserId,
 	} = parseFormData(formData);
-	const newPassport = await fetch(`https://id.purduehackers.com/api/new`, {
+	const newPassport = await fetch(await getNewIdApi(), {
 		method: "POST",
+		headers: await apiHeaders(),
 		body: JSON.stringify({
 			discord_id: stringUserId,
 			surname: trueSurname,
 			name: trueFirstName,
 			date_of_birth: trueDateOfBirth,
 			date_of_issue: trueDateOfIssue,
 			place_of_origin: placeOfOrigin,
+			ceremony_time: trueCeremonyTime,
 		}),
-	}).then((r) => r.json());
+	})
+		.then((r) => r.json())
+		.catch((err) => console.log("Error fetching new passport:", err));
 	return {
 		passportNumber: newPassport.id,
 	};
 }
+
+export async function getNewIdApi() {
+	if (process.env.PRODUCTION) {
+		return `https://id.purduehackers.com/api/new`;
+	}
+	return `https://id-git-staging-purdue-hackers.vercel.app/api/new`;
+}
+
+export async function apiHeaders() {
```

Comment:
Idk if stupid question, but why are these async?

Reply:
They fit best in that file, but "Server actions must be async functions" so they were async tagged

Reply:
lol TIL